azurerm/resource_arm_container_group_test.go

```diff
@@ -729,7 +813,7 @@ resource "azurerm_container_group" "test" {
         "node-name" = "acctestContainerGroup"
       }
     }
-  }
+	}
```

Comment:
Could we revert this?
```suggestion
  }
```